docs/contributor_guide/contributing_example_notebooks.rst

```diff
@@ -1,16 +1,16 @@
 Contributing example notebooks
 ------------------------------
 
-To contribute new example notebooks please submit a pull request. We have
-certain requirements that need to be satisfied to contribute a notebook. Those
-include:
+We'd love to collaborate with anyone interested in describing scenarios for using Fairlearn!
 
-* Clear structure
-* Appropriate framing of the problem while respecting that fairness is a
-  fundamentally sociotechnical challenge
-* Usage of at least some part of Fairlearn
+When discussing and critiquing examples, here are some lenses to consider:
 
-Good examples of existing notebooks that abide by these requirements are:
+Microrubric for critiques:
+"""""""""""""""""""""""""""""""""""
+1. **Deployment context**: Describes a real deployment context, not just a dataset.
+2. **Real harms**: Focuses on real harms to real people.  See `Blodget et al. (2020) <https://arxiv.org/abs/2005.14050>`_.
+3. **Sociotechnical**: Models the team's value that fairness as a sociotechnical challenge.    Avoids abstraction traps.  See `Selbst et al. (2020) <https://andrewselbst.files.wordpress.com/2019/10/selbst-et-al-fairness-and-abstraction-in-sociotechnical-systems.pdf>`_.
+4. **Substantiated**: Discusses trade-offs and compares alternatives. Describes why using particular Fairlearn functionalities makes sense.
+5. **For developers**: Speaks the language of developers and data scientists.  Considers real practitioner needs.  Fits within the lifecycle of real practioner work.  See `Holstein et al (2019) <https://arxiv.org/pdf/1812.05239.pdf>`_, `Madaio et al. (2020) <http://www.jennwv.com/papers/checklists.pdf>`_.
```

Comment:
Perhaps "Audience appropriate" rather than "For developers" - especially when the next sentence adds data scientists.

Reply:
That can mean anything, though. If I write a notebook for first graders and write it properly it's also audience appropriate, just perhaps not what we want here. How about "For practitioners"?

Reply:
@riedgar-ms Yeah, the main point is that the voice is not speaking to ML fairness researchers, as we talked about in the last Thursday meeting.

Also, I think we should focus revisions on what will help improve communication in the Thursday meetings.  The real goal isn't these words here, but to make an example notebook that will show what we mean to a broader audience far more powerfully than these words alone can :)

Reply:
True. This list is quite useful as an indication for new people joining the deep dive sessions, though.